template/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "source": "src/index.html",
   "scripts": {
-    "start": "parcel --open",
```

Comment:
This was required because codesandbox projects don't support this open flag - I believe it will be removed as part of #118 anyways